documentation/environment-operations-center-guide/secure-data-connectors/deploy-sdc-client.md

```diff
@@ -80,18 +80,177 @@ To deploy the secure data connector client on a Linux system, first unzip the *s
 - *appsettings.Production.json*
 - *RadiantLogic.OnPremisesAgentClient.Agent.exe*
 
-Open the *appsettings.Production.json* file and locate the `"AgentToken"` field. Enter the token copied from the *Data Connector Registration* dialog in Env Ops Center into the `"AgentToken"` field.
+Open the *appsettings.Production.json* file using an editor and locate the `"AgentToken"` field. Enter the token copied from the *Data Connector Registration* dialog in Env Ops Center into the `"AgentToken"` field.
 
-![image description](images/appsettings-token.png)
+![image description](images/linux-vi-appsettingsjson-file.png)
 
 If the client to be run on a network where proxy is setup, see the *Proxy Configuration for Linux* section in [run the secure data connector client under proxy network setup](deploy-sdc-client-in-proxy.md) guide.
 
+To deploy the sdc client on Linux, the server need to have the .NET core runtime. Depending on the Linux distribution, the installation steps can be found on the [official Microsoft website](https://learn.microsoft.com/en-us/dotnet/core/install/linux). To validate .NET is installed successfully, open a command line and run the dotnet command:
```

Comment:
Can you confirm if "dotnet" installation is required? I remember we copy all the dependent files as binary output to avoid dotnet dependency.